tests/queries/test_slippage_investigation.py

```diff
@@ -2,23 +2,28 @@
 
 from pprint import pprint
 
-from duneapi.api import DuneAPI
 from duneapi.types import DuneQuery, QueryParameter, Network
 
 from src.fetch.period_slippage import QueryType, slippage_query
 from src.models import AccountingPeriod
+from tests.db.pg_client import execute_dune_query, populate_db
 
 
 class TestDuneAnalytics(unittest.TestCase):
+    def setUp(self) -> None:
+        self.connection, self.cursor = populate_db()
+
+    def tearDown(self) -> None:
+        self.connection.close()
```

Comment:
In the future, we can also separate the populate DB script and put the truncation here (essentially clearing the DB for each test).